src/solvers/eigsolve.jl

```diff
@@ -0,0 +1,36 @@
+function eigsolve_updater(
+  init;
+  state!,
+  projected_operator!,
+  outputlevel,
+  which_sweep,
+  sweep_plan,
+  which_region_update,
+  region_kwargs,
+  updater_kwargs,
+)
+  default_updater_kwargs = (;
+    which_eigenvalue=:SR,
```

Comment:
I have a slight preference for calling this `which_eigval` since there is a function `eigvals` in `LinearAlgebra`.

Reply:
What about calling it `which` (the name of this keyword argument in `KrylovKit`)? Otherwise `which_eigval` is fine.

Reply:
From what I see, in `KrylovKit` `which` isn't a keyword argument, but a positional argument, so the name `which` isn't part of the `KrylovKit` interface (see https://jutho.github.io/KrylovKit.jl/stable/man/eig/#KrylovKit.eigsolve). My concern with calling it `which` is that it isn't clear what it is referring to.

Reply:
Nevermind, it's not a keyword arg in KrylovKit.